### PR TITLE
Updated keymap with better F positioning

### PR DESCRIPTION
**Pull request by pnikosis**

### CODE REVIEW

File: keyboards/planck/keymaps/dr_notsokind/keymap.c [hunk 1]
```diff
@@ -1,6 +1,5 @@
 // Layout picture at http://www.keyboard-layout-editor.com/#/gists/125febfad6960add078e6f14256539b6
 
-#pragma message "You may need to add LAYOUT_planck_grid to your keymap layers - see default for an example"
 #include "planck.h"
```
Comment (drashna) [inline keyboards/planck/keymaps/dr_notsokind/keymap.c]: Could you replace this:
```suggestion
#include QMK_KEYBOARD_H
```

File: keyboards/planck/keymaps/dr_notsokind/keymap.c [hunk 2]
```diff
@@ -1,6 +1,5 @@
...
 #include "action_layer.h"
```
Comment (drashna) [inline keyboards/planck/keymaps/dr_notsokind/keymap.c]: Could you remove all of these additional includes?  

```suggestion
```

They get included automatically, at various points in the built process, so they are redundant and not necessary here.

**Comment by drashna** [comment]
Additionally, this doesn't affect the pull request or the files in it, but I noticed that you've opened your pull request from your master branch, and that you have a number of merges in that branch.  

It's generally a good idea to not work from master, but create a new branch to work from and to only keep the master branch up to date. 

We have a good guide on how to do this, and the associated work flow here:
https://docs.qmk.fm/#/newbs_best_practices

But to emphasize, you don't need to close this PR, or even follow this link. It's a suggestion.

**Comment by pnikosis** [comment]
Hi drashna, just commited your suggestion. Sorry next time I will do it from a separate branch! (shame on me!)

**Comment by drashna** [comment]
Thanks!

**Pull request merged**